apps/email/style/compose_cards.css

```diff
@@ -22,6 +22,8 @@
 .cmp-bubble-container {
   width: calc(100% - 1rem);
   float: left;
+  display: inline-flex;
+  flex-wrap: wrap;
```

Comment:
The changes in here seem to cause an undesirable outcome for address entry. If the line is taken up by bubbles and there is just a little bit space left to enter the address, it does not break to a new line as the address gets longer. 

Picture here:

![screen](https://cloud.githubusercontent.com/assets/73359/5767254/7bfb7ab0-9cbc-11e4-972f-943b56ad22d1.png)
The "@dnd" breaks to a new line as the text gets longer for that space.

Not sure what to do about it yet though. Tried some quick things in dev tools. Ideally we could keep the older CSS styles as they are known to work, but when talking in IRC these were done so the screen reader could see the bubbles, is that correct? Or was it so it could see the input field? Maybe there is something else we could do here instead to get that to work and keep the line break behavior from before.